### ISSUE TITLE
Keyboard focus is not visible on the search filter if no value is selected in it.

### ISSUE THREAD

**Issue by shubham1121** [siemens/element#879]
### Prerequisites

- [x] I have read the [Contributing Guidelines](https://github.com/siemens/element/blob/main/CONTRIBUTING.md).
- [x] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### Affected component

<si-filtered-search>

### Summary

Keyboard focus is not visible on the search filter if no value is selected in it. Kindly refer the below attachment.

![Image](https://github.com/user-attachments/assets/7476a639-9183-4eb2-9a85-c4957484214b)

### Steps to reproduce

1. Navigate to the **si-filtered-search-basic** element page.  
2. Use the Tab key to move focus to the search section.  
3. Select an option from the dropdown.  
4. Press Tab twice.  
5. Press Shift + Tab and observe whether the focus is visible on the dropdown element.

### Code reproduction example

https://element.siemens.io/element-examples/#/overview/si-filtered-search/si-filtered-search-basic

### What is the current bug behavior

Focus is not visible on the dropdown element if no value is selected.

### What is the expected correct behavior

Keyboard focus should be clearly visible on the dropdown element when navigating using Tab or Shift + Tab, regardless of whether a value is selected.
